### ISSUE TITLE
app styling

### ISSUE THREAD

**Issue by FelineHuhn** [FelineHuhn/capstone-project-parenthood#20]
## Value Proposition

**As a** user
**I want to** to have a good looking app 
**so that** I can enjoy using it

## Acceptance criteria
- [x] the accessibility should be considered
- [x] the style is mobile first (iPhone SE)

## Tasks
- [x] open feature branch
- [x] add styling 
- [x] use styled components
- [x] check Level AA for accessibility criteria

## Complexity Estimation
M
